docs/azure-redis-cache.md

```diff
@@ -171,6 +175,19 @@
   cf unbind-service demoapp myrediscache
   ```
 
+## Updating
+
+  ```
+  cf update-service $service_instance_name -p $plan_name -c $updating_parameters
+  ```
+  For example:
+
+  ```
+  cf update-service myrediscache -p standardc0 -c '{"parameters":{"enableNonSslPort":true}}'
+  ```
+
+  You can update only the service plan, only the `enableNonSslPort` setting, or both. Note that, you can't change tier family and capacity in one single request per the limitation of Azure Redis service. For example, you can update from `basicc0` to `standardc0`. And you can't update from `basicc0` to `standardc1`. Instead, `basicc0` -> `standardc0` -> `standardc1` is OK.
```

Comment:
How about splitting them into two parts?
* Updating the service plan.
    Note that, you can't change tier family and capacity at the same time in one single request per the limitation of Azure Redis service. For example, you can update from `basicc0` to `standardc0`. But you can't update from `basicc0` to `standardc1`. You need to update from `basicc0` to `standardc0`, then from `standardc0` to `standardc1`.
* Updating the service properties.
  Currently, only the `enableNonSslPort` setting is supported to update.

Reply:
Good suggestion! Thanks, will use the statement you provided:).